mscore/pianoroll/pianoview.cpp

```diff
@@ -1902,10 +1902,10 @@ void PianoView::pasteNotesAtCursor()
 
       if (ms->hasFormat(PIANO_NOTE_MIME_TYPE)) {
             //Decode our XML format and recreate the notes
-            QByteArray data = ms->data(PIANO_NOTE_MIME_TYPE);
+            QByteArray dat = ms->data(PIANO_NOTE_MIME_TYPE);
```

Comment:
Here's another `dat` that could be renamed to `copiedNotes`.

Reply:
done